wordpress-mobile/WordPress-iOS
# Feature/placed pages pagination

This PR allows the pages list to load all the pages till the end, without using the infinite scroll feature as before.

Page List has a new `WPContentSyncHelper` called `PageContentSyncHelper` in charge to recursively call `syncMoreContent` till it has no more content.

**To test:**
- Open a blog and check if all the pages are loaded

@aerych do you mind to have a look since we already spoke about this?
